silverstripe/framework/SS-2015-006-1.yaml

```diff
@@ -0,0 +1,8 @@
+title:     SS-2015-006: XSS In GridField print
+link:      http://www.silverstripe.org/software/download/security-releases/ss-2015-006/
+cve:       ~
+branches:
+    3.1.x:
+        time:     2015-02-12 15:55:00
```

Comment:
Hello @xabbuh ,

I relying on the release date. Which blog post are you referring to? 

Spek